components/server/src/projects/projects-service.ts

```diff
@@ -119,6 +165,17 @@ export class ProjectsService {
         { name, slug, cloneUrl, teamId, appInstallationId }: CreateProjectParams,
         installer: User,
     ): Promise<Project> {
+        if (!(await this.auth.hasPermissionOnOrganization(installer.id, "create_project", teamId))) {
+            if (!this.auth.hasPermissionOnOrganization(installer.id, "read_info", teamId)) {
```

Comment:
missing await?

Reply:
Also goes to https://github.com/gitpod-io/gitpod/pull/18111/files#r1250981855

Reply:
Good catch! FWIW I could forget the `await` when declaring the local variable just like that. Would be great to have a listing rule that errors when using Promise in the context of a condition.

Reply:
There's https://eslint.org/docs/latest/rules/require-await which could work. Tho I do remember we had this already. Will have a look later today.